### ISSUE TITLE
List of requirements from users when they join

### ISSUE THREAD

**Issue by irnutsmurt** [avchikar97/SantaBot#52]
I wouldn't say this is an issue, more like a request for clarification for the users. When trying to start the the secret santa it errored out. My users had set their wishlist but no preferences. Its unclear when you're the user what are the requirements from them. My suggestion is a simple change in the text when they s!join. 

```
Welcome to the "channel name" Secret Santa! To complete your enrollment you'll need to input your wishlist URL and preferences (by DMing this bot) so your Secret Santa can send you something. 

Use s!setwishlisturl [wishlist urls separated by a space ] to set your wishlist URL (you may also add your mailing address).

Use s!setprefs [preferences separated by a space ] to set gift preferences for your Secret Santa. Put N/A if none.
```

After they're completed a nice message like "Congrats, you've finished enrollment in the secret santa!" I also suggest defaulting preferences to n/ to avoid erroring out (like I discovered this morning). If you prefer to have users input this then a a series of steps when they first join would help people complete it since most users don't fully read the instructions they're sent as I discovered when mine joined. Something like like the example below:

```
First we need your wishlist, please use s!setwishlisturl [wishlist urls separated by a space ] to set your wishlist URL (you may also add your mailing address).

s!setwishlist amazonurl/123 sesame street

GREAT! Now please specify what your preferences for your wishlist might be. Use s!setprefs [preferences separated by a space ] to set gift preferences for your Secret Santa. Put N/A if none.

s!setprefs Cheese Beans&Franks

Congrats, you're now officially enrolled in the "Channel Name" secret santa!
```
I think the easiest fix would just be to either not have the bot check for preferences or like I said earlier, setting the preferences to n/a by default

Thanks for the great work.

**Comment by avchikar97** [comment]
When you say `[wishlist urls separated by a space ]` does that mean you think the example provided is too verbose?

**Comment by irnutsmurt** [comment]
I'd say too vague. You need to assume people are too dumb since most on discord are not technical.

**Comment by avchikar97** [comment]
I added a commit to make preferences N/A by default and attempt to lead users through the sign-up process. Let me know if you think it needs more or have any other suggestions.

Just a heads-up, while I was testing it out, I found that this bot will need you to explicitly enable the Server Members Intent (looks like Discord changed their API so as of discord.py 1.5+ this is required) so that it can DM members about their partners when the Secret Santa is started. I've updated the README to include this, just wanted to let you know.
